### DISCUSSION TITLE
Enter key doesn't send message

### DISCUSSION THREAD

**Discussion by wanestar** [open-webui/open-webui#1922] [General]
The latest update changes the Enter key function to only move to the next line instead of sending the message. 

While this is interesting, I find using Shift-Enter to send messages works well. However, I would greatly prefer it if the Enter key were used for sending messages, as it feels more intuitive. 

Perhaps adding a setting for this would be a good solution?

Thanks

**Comment by arsaboo** [comment]
Yes, that is the behavior across all UIs. Enter to send and Shift+Enter for the next line.

**Comment by wanestar** [comment]
Hmm, Well it seems to have broke for me in v0.1.123, Enter only moves to next line now

**Reply by tjbck** [reply]
Are you using a touch device by chance?

**Reply by wanestar** [reply]
No. just updated my windows container. Worked fine on last release.

**Reply by tjbck** [reply]
Just merged a refac that would most likely resolve your issue. Let me know if the issue persists!

**Reply by wanestar** [reply]
Yep! That seems to have fixed it. Thanks for the fast support.

**Comment by fultonm** [comment]
This is happening for me. Shift+Enter and Enter both add a newline.

**Reply by veivataan** [reply]
This is happening to me too! But it's unpredictable: if i open in a new tab it can fix it sometimes.
